src/lib/utils/__tests__/spotify.js

```diff
@@ -0,0 +1,8 @@
+/*eslint-env jest*/
+var spotify = require('spotify-node-applescript')
```

Comment:
And here https://github.com/basedjux/hyperline/blob/6e33f6e85670ab564a92e4e2e954bd84b57a553e/src/lib/plugins/spotify.js#L4

This can probably be changed though:
`import spotify from 'spotify-node-applescript'`